spring-cloud-consul-discovery/src/main/java/org/springframework/cloud/consul/discovery/ConsulServiceInstance.java

```diff
@@ -25,23 +26,29 @@
 
 import org.springframework.cloud.client.DefaultServiceInstance;
 import org.springframework.core.style.ToStringCreator;
+import org.springframework.util.StringUtils;
 
 import static org.springframework.cloud.consul.discovery.ConsulServerUtils.findHost;
 
 public class ConsulServiceInstance extends DefaultServiceInstance {
 
 	private HealthService healthService;
 
-	public ConsulServiceInstance(HealthService healthService, String serviceId) {
```

Comment:
This is a public API so existing constructor signatures will need to remain with a default value.

Reply:
There is already a constructor with default merge value as `false`.

I just moved the default constructor to be the first in the file.